src/stackmap_checker/stmap.c

```diff
@@ -0,0 +1,180 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <err.h>
+#include "stmap.h"
+
```

Comment:
What does this file do? Can we add a comment?

Reply:
I've documented the non-obvious functions defined in this file in `stmap.h`. I'll also add a brief description of the whole thing.